docs/source/graphbolt/ondisk-dataset.rst

```diff
@@ -0,0 +1,829 @@
+.. _graphbolt-ondisk-dataset:
+
+Prepare dataset
+===============
+
+**GraphBolt** provides the ``OnDiskDataset`` class to help user organize graph
+strucutre, feature data and tasks. ``OnDiskDataset`` is designed to efficiently
+handle large graphs that do not fit into memory by storing them on disk.
```

Comment:
For graphs that can fit into memory, OnDiskDataset is still the recommended way to organize it before loading to memory.
```suggestion
**GraphBolt** provides the ``OnDiskDataset`` class to help user organize plain data of graph structure, feature and tasks . ``OnDiskDataset`` is also designed to efficiently
handle large feature that do not fit into memory by storing them on disk.
```